catroid/src/main/res/values/strings.xml

```diff
@@ -1757,6 +1757,8 @@ needs read and write access to it. You can always change permissions through you
     <string name="formula_editor_function_join_parameter">(\'hello\',\' world\')</string>
     <string name="formula_editor_function_regex">regular expression</string>
     <string name="formula_editor_function_regex_parameter">(\' an? ([^ .]+)\',\'I am a panda.\')</string>
+    <string name="formula_editor_function_regex_assistant">\t\t\t\t\t Regular expression
```

Comment:
These English strings are the basis for our translators, who use our CrowdIn https://translate.catrobat.org/ system and who are not developers, and who cannot see or control the layout of the strings in the app. It would therefore be better if these indents are added through code. 

Please also make sure that the indentation looks fine in right to left languages, i.e., is added on the right side for Arabic or Ivrit.